x/emissions/keeper/inference_synthesis/network_inferences.go

```diff
@@ -78,6 +78,29 @@ func calcNetworkInferencesMultipleByMedian(
 		return nil, errorsmod.Wrap(err, "while calculating median")
 	}
 
+	// Compute one-out average for each inferer
+	oneOutInfererValues := make([]*emissions.WithheldWorkerAttributedValue, 0, len(inferences.Inferences))
+	for i, inf := range inferences.Inferences {
+		// Build slice without the current inference
+		without := make([]alloraMath.Dec, 0, len(inferenceValues)-1)
+		for j, val := range inferenceValues {
+			if i != j {
+				without = append(without, val)
+			}
+		}
```

Comment:
```suggestion
        without := append([]alloraMath.Dec(nil), inferenceValues[:i]...)
		without = append(without, inferenceValues[i+1:]...)
```

This can be simplified

Reply:
Thanks! This PR is abandoned though - please review this other one instead: https://github.com/allora-network/allora-chain/pull/871